lib/kafka_ex/server.ex

```diff
@@ -209,21 +214,21 @@ defmodule KafkaEx.Server do
     Enum.each(state.brokers, fn(broker) -> KafkaEx.NetworkClient.close_socket(broker.socket) end)
   end
 
-  defp update_consumer_metadata(state, consumer_group), do: update_consumer_metadata(state, consumer_group, 3, 0)
+  defp update_consumer_metadata(state), do: update_consumer_metadata(state, 3, 0)
```

Comment:
What's the significance of the 3 here?  Looks like a number of tries?  Should it be a module attribute?